config/manager/kustomization.yaml

```diff
@@ -14,7 +14,7 @@ apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 images:
 - name: controller
-  newName: quay.io/netobserv/network-observability-operator
-  newTag: 1.0.2
+  newName: quay.io/acmenezes/network-observability-operator
+  newTag: dev
```

Comment:
this will have to be rollbacked at some point (you can keep it as long as it's a draft if that's easier for you)

Reply:
For sure. I won't let it there. I actually shouldn't have committed it.

Reply:
It's cleaned up now.